src/blocks/gallery-carousel/deprecated.js

```diff
@@ -1181,6 +1181,188 @@ const deprecated =
+					[ `has-margin-right-${ gutter }` ]: gutter > 0,
+					[ `has-margin-right-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+				}
+			);
+
+			const thumbnailClasses = ( index ) => {
+				return classnames(
+					'wp-block-coblocks-gallery-carousel-thumbnail',
+					`wp-block-coblocks-gallery-carousel-thumbnail-${ index }`,
+					{
+						[ `has-margin-left-${ gutter }` ]: gutter > 0,
+						[ `has-margin-left-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+						[ `has-margin-right-${ gutter }` ]: gutter > 0,
+						[ `has-margin-right-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+					}
+				);
+			};
+
+			const thumbnailContainerClasses = classnames(
+				'wp-block-coblocks-gallery-carousel-thumbnail-pagination',
+				{
+					[ `has-margin-top-${ gutter }` ]: gutter > 0,
+					[ `has-margin-top-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+				}
+			);
+
+			const captionClasses = classnames(
+				'coblocks-gallery--caption',
+				'coblocks-gallery--primary-caption', {}
+			);
+
+			const swiperClasses = classnames(
+				'has-carousel',
+				`has-carousel-${ gridSize }`,
+				'swiper-container',
+				{
+					'has-aligned-cells': alignCells,
+					'has-responsive-height': responsiveHeight,
+					[ navForClass ]: thumbnails,
+				}
+			);
+
+			const swiperStyles = {
+				height: height ? `${ height }px` : undefined,
+			};
+
+			const uuid = '12345';
+
+			const swiperSizing = {
+				lrg: 2,
+				med: 4,
+				sml: 5,
+				xlrg: 1,
+			};
+
+			const swiperOptions = {
+				alignCells,
+				autoPlay,
+				autoPlaySpeed,
+				draggable,
+				freeScroll,
+				loop,
+				navigation: prevNextButtons,
+				pageDots,
+				pauseHover,
+				responsiveHeight,
+				slidesPerView: swiperSizing[ gridSize ],
+				thumbnails,
+				uuid,
+			};
+
+			return (
+				<div aria-label={ __( `Carousel Gallery`, 'coblocks' ) } >
+					<div className={ innerClasses }>
+						<div className={ swiperClasses } data-swiper={ JSON.stringify( swiperOptions ) } style={ swiperStyles } >
+							<div className="swiper-wrapper">
+								{ images.map( ( image, index ) => {
+									return (
+										<div className="swiper-slide" key={ index }>
+											<div
+												className="coblocks-gallery--item"
+												role="button"
+												tabIndex={ index }
+											>
+												<figure className={ figureClasses }>
+													<img
+														alt={ image.alt }
+														className={ image.id ? `wp-image-${ image.id }` : null }
+														data-id={ image.id }
+														data-link={ image.link }
+														src={ image.url }
+													/>
+												</figure>
+												<RichText.Content className={ captionClasses } tagName="figcaption" value={ image.caption } />
+											</div>
+										</div>
+									);
+								} ) }
+							</div>
+							{ prevNextButtons && (
+								<>
+									<button className={ `nav-button__prev` } id={ `${ uuid }-prev` } >
```

Comment:
If someone has two carousels on the page, the accessibility tests will still fail.

Reply:
You are right. These ID's aren't used by use so I don't think they actually need to be present.